src/pages/FrontendMentor/SkIPAddress/index.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { ChangeEvent, useEffect, useState } from 'react'
+import Background from '../../../components/SkIPAddress/Background'
+import ContentBox from '../../../components/SkIPAddress/ContentBox'
+import {
+  BoxSeparator,
+  DisplayContainer,
+  InputContainer,
+  MainContainer,
+  MyMap,
+  StyledButton,
+  StyledHeader,
+  StyledInput,
+} from './styles/SkIPAddress.styled'
+import { TileLayer, Marker, useMap } from 'react-leaflet'
+interface RequestData {
+  location?: Location
+  ip: string
+  isp: string
+}
+interface Location {
+  city: string
+  country: string
+  region: string
+  lat: number
+  lng: number
+  postalCode: string
+  timezone: string
+}
```

Comment:
i think try n use type instead of interface! https://stackoverflow.com/a/54101543/16008592

Reply:
It says prefer interface over type eh

Reply:
47827e1